chapters/tests/sections/manual_tests.tex

```diff
@@ -0,0 +1,58 @@
+\section{Manual testing} \label{sec:manual_testing}
+Manual testing is responsible for testing the game as a whole.
+It tests the results and not particular methods or parts of the code.
+Manual tests scenarios were created that describe in detail what should happen in the game after some actions are performed.
+This tests cover the entirety of the application and can be viewed at \url{https://github.com/Non-Euclidean-World/HyperTesting}.
+The tests are written as Markdown files that have references to image and video files that show the expected behavior of the game.
```

Comment:
maybe write something about converting this stuff to pdfs but that's optional don't know if we want to write about it because it was pretty pointless requirement in the first place to have in pdf

Reply:
I don't think we will ever convert them to pdf again. it was only used for deliverable4